packer/ansible/roles/images/tasks/main.yml

```diff
@@ -20,3 +20,11 @@
    - monorail-efi32-snponly.efi
    - monorail-efi64-snponly.efi
    - undionly.kpxe
```

Comment:
remove 'undionly.kpxe'? now on-http master has update undionly.kpxe to undionly.kkpxe https://github.com/RackHD/on-http/blob/master/data/profiles/install-esx.ipxe#L5